### ISSUE TITLE
Local battery template overrides experiment's style.css

### ISSUE THREAD

**Issue by earcanal** [expfactory/expfactory-battery#70]
I'm not sure how the templates are generated, but the installed `localbattery.html` (2.5.47) contains:

    <head>
        ...
        <link rel='stylesheet' href='static/css/default_style.css' type="text/css">
    </head>

This refers to `default_style.css` *after* an experiment's `style.css` making it awkward to override styles (e.g. experiment background colour).  As the canonical `config.json` includes `default_style.css` *before* `style.css` it looks like the reference in the template should be removed.

**Comment by vsoch** [comment]
Agreed, the user style needs to come first. We can't assume, however that the `default_style.css` would be included by the user in the config.json. Would it suffice to just move around the ordering so user style takes preference?

**Comment by earcanal** [comment]
Yeah, it's probably a better solution if the user style is the last thing in the template.

**Comment by vsoch** [comment]
This sounds good to me. Would you like to PR with the change?

**Comment by earcanal** [comment]
I would, but I can't figure out how that template is built.

**Comment by vsoch** [comment]
I believe it's here:

https://github.com/expfactory/expfactory-battery/blob/master/templates/exp.html

**Comment by earcanal** [comment]
How is that used to build `localbattery.html`?

**Comment by vsoch** [comment]
Ah apologies - it isn't! I think we want this one from the other repository:

https://github.com/expfactory/expfactory-python/blob/master/expfactory/templates/localbattery.html

Apologies, it's been a while!

**Comment by earcanal** [comment]
:) No worries.  Might be easier for you to commit than process a PR for such a small change?

**Comment by vsoch** [comment]
good idea :)

All set : https://github.com/expfactory/expfactory-python/blob/master/expfactory/templates/localbattery.html

Give it a try!

**Comment by earcanal** [comment]
Something not quite right with that.  Looks like `default_style.css` is being injected a bit too early (in relation to `config.json` includes, which puts it after `jspsych.css`).

**Comment by vsoch** [comment]
I haven't worked on this project in years, and I'm trying to help out because my colleagues are silent. If you would be able to, it would be great for you to test this and do a PR since others will likely have the same trouble. Thanks!

**Comment by earcanal** [comment]
I've _never_ worked on it, and Python isn't one of my strengths.  I think your commit solves the immediate issue, so I'll create a PR with that and some tech debt comments.

**Comment by earcanal** [comment]
PR submitted.
